newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -71,7 +71,7 @@ function getCandidateSlotProps(startTime, duration, minHour, maxHour) {
   const endTime = toMoment(startTime, DEFAULT_TIME_FORMAT)
     .add(duration, 'm')
     .format(DEFAULT_TIME_FORMAT);
-  return getSlotProps(startTime, endTime, minHour, maxHour, duration);
+  return getSlotProps(startTime, endTime, minHour, maxHour);
```

Comment:
I believe this `duration` argument was not being captured by the function so I removed it

Reply:
Indeed, thanks!